mkdocs.yml

```
@@ -212,334 +212,334 @@ plugins:
   - redirects:
       redirect_maps:
         # redirect removed pages to sensible alternative
-        easybuild-v5/backwards-incompatible-changes/: easybuild-v5/index.md
-        easybuild-v5/changes-in-default-configuration.html: easybuild-v5/changes.md
-        easybuild-v5/overview-of-changes/: easybuild-v5/index.md
-        easybuild-v5/proposed-changes/: easybuild-v5/index.md
+        easybuild-v5/backwards-incompatible-changes.md: easybuild-v5/index.md
+        easybuild-v5/changes-in-default-configuration.md: easybuild-v5/changes.md
+        easybuild-v5/overview-of-changes.md: easybuild-v5/index.md
```

@branfosj Shouldn't this be:
```suggestion
        easybuild-v5/overview-of-changes/index.md: easybuild-v5/index.md
```

Likewise for other redirects for `.../`